contracts/FlowToken.cdc

```diff
@@ -167,32 +241,32 @@ pub contract FlowToken: FungibleToken {
         }
     }
 
-    init(adminAccount: AuthAccount) {
+    init() {
         self.totalSupply = 0.0
 
         // Create the Vault with the total supply of tokens and save it in storage
         //
         let vault <- create Vault(balance: self.totalSupply)
-        adminAccount.save(<-vault, to: /storage/flowTokenVault)
+        self.account.save(<-vault, to: /storage/flowTokenVault)
 
         // Create a public capability to the stored Vault that only exposes
         // the `deposit` method through the `Receiver` interface
         //
-        adminAccount.link<&FlowToken.Vault{FungibleToken.Receiver}>(
+        self.account.link<&FlowToken.Vault{FungibleToken.Receiver, FungibleToken.Balance, MetadataViews.Resolver}>(
             /public/flowTokenReceiver,
```

Comment:
Are you able to resolve the FTVaultData metadata view and use its path values here and in other places. That way you can guarantee they're the same, and anyone who pokes around on the flow token contract to find paths will see an example of how to resolve that metadata

Reply:
Sorry, I'm a little confused about what you're suggesting. You're just saying that we should resolve the view in the `init` method just to show an example of how it is done on a common contract? Not anything practical?

I think that is probably a good idea, so I can include that